lib/rules/indent.js

```diff
@@ -1385,6 +1373,15 @@ module.exports = {
                 if (astUtils.isSemicolonToken(lastToken)) {
                     offsets.ignoreToken(lastToken);
                 }
+
+                if (options.VariableDeclarator[node.kind] === "first") {
+                    addElementListIndent(
+                        node.declarations,
+                        sourceCode.getFirstToken(node),
+                        sourceCode.getLastToken(node),
```

Comment:
The use of `sourceCode.getLastToken` here seems a bit strange. Normally, when `addElementListIndent`, the last token is indented the same way as the first token (e.g. in `[foo, bar, baz]`, the `]` is indented the same amount as the `[`.

It might not be necessary to change this because the last token would be indented properly later on (e.g. in the `ArrayExpression` listener). However, can you add tests for cases like this?

```js
let foo = 1,
    bar = 2,
    baz
```

```js
let
    foo
```

```js
let foo = 1,
    bar =
    2
```

Reply:
Do you expect the cases above are reporting warnings or not?